libr/anal/var.c

```diff
@@ -1435,13 +1435,13 @@ R_API void r_anal_var_list_show(RAnal *anal, RAnalFunction *fcn, int kind, int m
 					eprintf ("Register not found");
 					break;
 				}
-				anal->cb_printf ("afv%c %s %s %s @ 0x%"PFMT64x "\n",
+				anal->cb_printf ("\"afv%c %s %s %s @ 0x%"PFMT64x "\"\n",
```

Comment:
@ doesnt work if you quote the command

Reply:
In this case is probably better to use s and s- and (or keep the @ and) filter the name and type to not contain special chars (see rnamefilter api)